changelog/@unreleased/pr-5128.v2.yml

```diff
@@ -0,0 +1,7 @@
+type: feature
+feature:
+  description: AtlasDb can now be configured by its clients to use custom request
+    batcher for batching requests to leaderTime api on Timelock. In the absence of
+    custom batcher, we fall back to legacy leaderTime request coalescing batcher.
```

Comment:
nit: probably want articles i.e. on line 3, configured by its clients to use *a* custom request batcher; line 5, in the absence of *a* custom batcher, we fall back to *a* legacy leaderTime request coalescing batcher.